unitTests/ui/MLFacet/MLFacetValues/MLFacetValueTest.ts

```diff
@@ -60,9 +66,37 @@ export function MLFacetValueTest() {
       expect(mLFacetValue.equals(value)).toBe(true);
     });
 
-    // TODO: tests
-    // formattedCount
-    // valueCaption
-    // render (only check that is doesn't throw)
+    it(`when getting formattedCount with a non zero value
+      it should return a string in the Globalize format`, () => {
+      expect(mLFacetValue.formattedCount).toBe(Globalize.format(mLFacetValue.numberOfResults, 'n0'));
+    });
+
+    it(`when getting formattedCount with a zero value
+      it should return an empty string`, () => {
+      mLFacetValue.numberOfResults = 0;
+      expect(mLFacetValue.formattedCount).toBe('');
+    });
+
+    it(`when using the valueCaption option with a function
```

Comment:
Should the test ensures we are passing the original facet value to the function? Atm, we return `allo` all the time.

Maybe something like:

```
{ valueCaption: originalValue => `${originalValue}_caption`}
```

Reply:
passing a function simply should not work, not supported

Reply:
Oh, I misread and misunderstood the test 👍